src/Main.cpp

```diff
@@ -90,6 +91,7 @@ int main(int argc, char **argv)
 
 	if (ModeListBasic || ModeListHex || ModeListDams || ModeDisaFile || ModeListAscii)
 	{
+		NoOptionSet = true;
```

Comment:
Are you sur you do not want to put NoOptionset to false ? By putting it to true, the catalog is displayed even when a file is listed.

Personally I would put false.